corehq/toggles.py

```diff
@@ -56,7 +56,8 @@
     css_class='default',
     description="These are tools for our engineering team to use to manage the product",
 )
-ALL_TAGS = [TAG_CUSTOM, TAG_DEPRECATED, TAG_PRODUCT, TAG_SOLUTIONS, TAG_INTERNAL]
+# Order roughly corresponds to how much we want you to use it
+ALL_TAGS = [TAG_SOLUTIONS, TAG_PRODUCT, TAG_CUSTOM, TAG_INTERNAL, TAG_DEPRECATED]
```

Comment:
I was surprised that product was after solutions here.

Reply:
Yeah, I was surprised about this too - going in to this I had a vague idea of both Product and Solutions flags being on similar footing, with Product perhaps a bit ahead.  Talking with Andrea and Dev, it sounds like the "Product" tag is intended for testing in-dev features, whereas I believe anything we want to promote use of that'll be a feature flag indefinitely should be tagged Solutions.

I do think we need to clarify that point a bit and probably reorganize some existing flags, though.